unitTest/@testFEMComputationalFluidStructureInteractionAnalysis/testFluidStructureInteractionSpringMass.m

```diff
@@ -36,7 +36,7 @@ function testFluidStructureInteractionSpringMass(testCase)
 %% 0. Read input
 
 % Define absolute tolerance
-absTol = 1e-15;
+absTol = 1e-8;
```

Comment:
Please revert the changes in the unit tests, I would like to fix them myself under Windows.